[2024-12-16T18:31:38.902+0000] {processor.py:157} INFO - Started process (PID=32) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:31:38.904+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:31:38.915+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:31:38.909+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:31:38.940+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:31:38.928+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:31:38.942+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:31:39.072+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.184 seconds
[2024-12-16T18:32:09.365+0000] {processor.py:157} INFO - Started process (PID=42) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:32:09.366+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:32:09.366+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:32:09.366+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:32:09.370+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:32:09.369+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:32:09.370+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:32:09.380+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.017 seconds
[2024-12-16T18:33:23.486+0000] {processor.py:157} INFO - Started process (PID=31) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:33:23.489+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:33:23.492+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:33:23.491+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:33:23.512+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:33:23.507+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:33:23.514+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:33:23.565+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.100 seconds
[2024-12-16T18:33:53.929+0000] {processor.py:157} INFO - Started process (PID=41) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:33:53.929+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:33:53.930+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:33:53.930+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:33:53.932+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:33:53.931+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:33:53.933+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:33:53.941+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.014 seconds
[2024-12-16T18:34:24.497+0000] {processor.py:157} INFO - Started process (PID=51) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:34:24.497+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:34:24.498+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:34:24.498+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:34:24.500+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:34:24.500+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:34:24.500+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:34:24.509+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.014 seconds
[2024-12-16T18:34:55.249+0000] {processor.py:157} INFO - Started process (PID=61) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:34:55.250+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:34:55.250+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:34:55.250+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:34:55.252+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:34:55.252+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:34:55.253+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:34:55.262+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.014 seconds
[2024-12-16T18:35:25.966+0000] {processor.py:157} INFO - Started process (PID=71) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:35:25.967+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:35:25.967+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:35:25.967+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:35:25.969+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:35:25.969+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:35:25.969+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:35:25.978+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.014 seconds
[2024-12-16T18:35:56.732+0000] {processor.py:157} INFO - Started process (PID=81) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:35:56.733+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:35:56.733+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:35:56.733+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:35:56.735+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:35:56.735+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:35:56.736+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:35:56.744+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.014 seconds
[2024-12-16T18:36:27.610+0000] {processor.py:157} INFO - Started process (PID=91) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:36:27.610+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:36:27.610+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:36:27.610+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:36:27.613+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:36:27.612+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:36:27.613+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:36:27.624+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.016 seconds
[2024-12-16T18:36:58.323+0000] {processor.py:157} INFO - Started process (PID=101) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:36:58.323+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:36:58.324+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:36:58.323+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:36:58.326+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:36:58.326+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:36:58.326+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:36:58.335+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.014 seconds
[2024-12-16T18:37:29.123+0000] {processor.py:157} INFO - Started process (PID=111) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:37:29.123+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:37:29.124+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:37:29.124+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:37:29.127+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:37:29.126+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:37:29.127+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:37:29.139+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.018 seconds
[2024-12-16T18:38:00.154+0000] {processor.py:157} INFO - Started process (PID=121) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:38:00.154+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:38:00.155+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:38:00.155+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:38:00.159+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:38:00.157+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:38:00.159+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:38:00.175+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.023 seconds
[2024-12-16T18:38:31.017+0000] {processor.py:157} INFO - Started process (PID=131) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:38:31.018+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:38:31.018+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:38:31.018+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:38:31.021+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:38:31.020+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:38:31.021+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:38:31.030+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.014 seconds
[2024-12-16T18:39:01.805+0000] {processor.py:157} INFO - Started process (PID=141) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:39:01.806+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:39:01.806+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:39:01.806+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:39:01.809+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:39:01.808+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:39:01.809+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:39:01.821+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.017 seconds
[2024-12-16T18:39:32.578+0000] {processor.py:157} INFO - Started process (PID=151) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:39:32.579+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:39:32.579+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:39:32.579+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:39:32.581+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:39:32.581+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:39:32.582+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:39:32.591+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.014 seconds
[2024-12-16T18:40:03.326+0000] {processor.py:157} INFO - Started process (PID=161) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:40:03.327+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:40:03.327+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:40:03.327+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:40:03.330+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:40:03.329+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:40:03.330+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:40:03.464+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.140 seconds
[2024-12-16T18:43:35.353+0000] {processor.py:157} INFO - Started process (PID=32) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:43:35.354+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:43:35.356+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:43:35.356+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:43:35.375+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:43:35.371+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:43:35.375+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:43:35.406+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.054 seconds
[2024-12-16T18:44:05.704+0000] {processor.py:157} INFO - Started process (PID=42) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:44:05.705+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:44:05.705+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:44:05.705+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:44:05.708+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:44:05.707+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:44:05.709+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:44:05.718+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.016 seconds
[2024-12-16T18:44:36.311+0000] {processor.py:157} INFO - Started process (PID=52) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:44:36.311+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:44:36.312+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:44:36.312+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:44:36.314+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:44:36.313+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:44:36.314+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:44:36.323+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.014 seconds
[2024-12-16T18:45:53.861+0000] {processor.py:157} INFO - Started process (PID=32) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:45:53.867+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:45:53.868+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:45:53.868+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:45:53.881+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:45:53.877+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:45:53.884+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:45:53.984+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.131 seconds
[2024-12-16T18:46:35.883+0000] {processor.py:157} INFO - Started process (PID=32) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:46:35.884+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:46:35.884+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:46:35.884+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:46:35.888+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:46:35.887+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:46:35.888+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:46:35.919+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.040 seconds
[2024-12-16T18:47:06.100+0000] {processor.py:157} INFO - Started process (PID=42) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:47:06.101+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:47:06.101+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:47:06.101+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:47:06.104+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:47:06.103+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:47:06.104+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:47:06.113+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.015 seconds
[2024-12-16T18:47:36.750+0000] {processor.py:157} INFO - Started process (PID=52) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:47:36.751+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:47:36.752+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:47:36.752+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:47:36.756+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:47:36.755+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:47:36.756+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:47:36.766+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.019 seconds
[2024-12-16T18:48:07.731+0000] {processor.py:157} INFO - Started process (PID=62) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:48:07.732+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:48:07.732+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:48:07.732+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:48:07.735+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:48:07.734+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:48:07.735+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:48:07.743+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.014 seconds
[2024-12-16T18:48:38.477+0000] {processor.py:157} INFO - Started process (PID=72) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:48:38.478+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:48:38.478+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:48:38.478+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:48:38.482+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:48:38.481+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:48:38.483+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:48:38.495+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.020 seconds
[2024-12-16T18:49:09.016+0000] {processor.py:157} INFO - Started process (PID=82) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:49:09.017+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:49:09.017+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:49:09.017+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:49:09.021+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:49:09.020+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:49:09.022+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:49:09.033+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.019 seconds
[2024-12-16T18:49:39.999+0000] {processor.py:157} INFO - Started process (PID=92) to work on /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:49:40.000+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/spark_pubsub_test.py for tasks to queue
[2024-12-16T18:49:40.000+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:49:40.000+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:49:40.003+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:49:40.002+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/spark_pubsub_test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/spark_pubsub_test.py", line 1, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2024-12-16T18:49:40.004+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/spark_pubsub_test.py
[2024-12-16T18:49:40.013+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/spark_pubsub_test.py took 0.015 seconds
